network-api/networkapi/wagtailpages/templates/wagtailpages/blog_page.html

```diff
@@ -0,0 +1,24 @@
+{% extends "./modular_page.html" %}
+
+{% load wagtailcore_tags mini_site_tags %}
+
+{% block pre_footer_classes %}{% if singleton_page %}col-12 col-lg-10 offset-lg-1{% else %}col-md-12{% endif %}{% endblock %}
```

Comment:
Is the `singleton_page` check necessary here?

Reply:
To rephrase that question, can blog pages ever have children? If not, then the template can be simplified.

Reply:
@alanmoo ^ ?

Reply:
I don't believe they should be able to have children. Let's make sure the option doesn't exist (something like `subpage types` needs to get set somewhere?) and simplify.

Reply:
https://github.com/mozilla/foundation.mozilla.org/search?q=pre_footer_classes&unscoped_q=pre_footer_classes

<img width="738" alt="image" src="https://user-images.githubusercontent.com/2896608/57336835-35d86180-70dc-11e9-9bfb-899af6db9dd0.png">

It seems like `pre_footer_classes` block isn't doing anything. I'll update `blog_page.html` and file a ticket to remove `{% block pre_footer_classes %}` for good